src/input.tpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <limits>
+#include <string>
+#include "../include/input.h"
```

Comment:
No need to include `input.h` here. `input.tpp` is already include here https://github.com/HEIGVD-PRG1-F-2022/prg1f-io/blob/1d656f333abc34fc8ae55a4c952bf4a2bd6568ae/include/input.h#L26

Reference: https://stackoverflow.com/questions/495021/why-can-templates-only-be-implemented-in-the-header-file

Reply:
Done